lang/java/reef-common/src/main/java/org/apache/reef/driver/restart/DriverRestartManagerImpl.java

```diff
@@ -45,30 +45,39 @@
 
   private boolean restartBegan;
   private boolean restartCompleted;
+  private boolean isRestartSet;
```

Comment:
Also, the two new ones only seem to be used once, and could be replaced with a constant?